lib/trino-hdfs/src/main/java/io/trino/hdfs/s3/TrinoS3FileSystem.java

```diff
@@ -924,17 +925,12 @@ private static boolean isHadoopFolderMarker(S3ObjectSummary object)
         return object.getKey().endsWith("_$folder$");
     }
 
-    /**
```

Comment:
I think I'd still leave the comment, that still applies no?

Reply:
Oh I see you moved it to UnrecoverableIOException. Seems reasonable but I'd keep some of the examples like Forbidden in the comment in `UnrecoverableIOException`. That looks to be missing.